list/LIST_HEADING_README.md

```diff
@@ -0,0 +1,162 @@
+# List (Heading) Mapping
+
+## XML HL7 > JSON FHIR
+
+A List (Heading) is created from a [List (Topic)](./LIST_TOPIC_README.md) member (referred to as `Topic`) and a `CompoundStatement` within the same `ehrComposition` which has a classCode of `CATEGORY`
+This represents the SOAP heading sections which contain clinical record entries.
```

Comment:
I'm confused, does SOAP here mean this [SOAP](https://en.wikipedia.org/wiki/SOAP)?

Reply:
Yes... I believe so, but I can find no other reference to SOAP heading sections other than in the definition of this section.  I'm guessing it's somehow related to this but theres no examples which seem to provide any useful information

![image](https://github.com/NHSDigital/patient-switching-adaptors-mapping-documentation/assets/88717465/4d603a8a-55e5-4c6f-b71c-79427bcbffe4)

Reply:
SOAP appears to be relating to a format of medical records note taking format; These stand for: Subjective, Objective, Analysis, Plan.  A little confusing when framed from a technical perspective!

Reply:
Have updated to explain this